src/generate_leaderboards.py

```diff
@@ -450,11 +420,12 @@ def extract_model_metadata(results: list[dict]) -> dict[str, dict]:
                 merge=record.get("merge", False),
             )
         )
-        if record["dataset"] == "speed":
-            metadata_dict[model_id]["speed"] = record["results"]["total"]["test_speed"]
 
-        version = record.get("euroeval_version", "<9.2.0@@0")
-        if "@" not in version:
+        version = record.get(
+            "scandeval_version", "<9.2.0"
+        )  # version was not safed before 9.2.0
+
```

Comment:
The `get` is not required anymore, since this is dealt with in the `process_results` script:

```suggestion
        version = record["euroeval_version"]
```

Reply:
we still need the get (at least certain files don't have the version)